orderer/consensus/etcdraft/storage_test.go

```diff
@@ -36,7 +37,8 @@ var (
 func setup(t *testing.T) {
 	logger = flogging.NewFabricLogger(zap.NewExample())
 	ram = raft.NewMemoryStorage()
-	dataDir = t.TempDir()
+	dataDir, err = ioutil.TempDir("", "etcdraft-")
```

Comment:
why the revert?

Reply:
I couldn't find the reason why this is causing the CI unit test to fail. The CI passes when I revert back to `ioutil.TempDir`. 

Should we just continue with `t.TempDir`? What do you think?

Reply:
I'm fine with how it is right now

Reply:
It seems like the CI unit test _can_ pass with `t.TempDir`... https://github.com/Juneezee/fabric/actions/runs/3046565807/jobs/4909614997

Reply:
I suspect that some of those unit tests are just a bit flakey in the build environment, and that your change is not the cause of any failure. @yacovm will know better than me though, and should be able to re-trigger failed builds if they are likely to be caused by intermitted test failures.

Reply:
I don't recall storage test to be flaky. chain test is flaky, but that's about it